wire-ios-sync-engine/Source/Data Model/Conversation+AccessMode.swift

```diff
@@ -77,11 +82,17 @@ extension ZMConversation {
         let request = WirelessRequestFactory.fetchLinkRequest(for: self, apiVersion: apiVersion)
         request.add(ZMCompletionHandler(on: managedObjectContext!) { response in
             if response.httpStatus == 200,
-               let uri = response.payload?.asDictionary()?[ZMConversation.TransportKey.uri] as? String {
-                completion(.success(uri))
+               let payloadDict = response.payload?.asDictionary() {
+                let hasPassword = payloadDict["has_password"] as? Bool ?? false
```

Comment:
Can we add "has_password" to the `ZMConversation.TransportKey`?

Reply:
Done!